### PR TITLE
refactor: clean up parameters of excludePlaylist

### PR DESCRIPTION
**Pull request by gesinger**
Rename the parameters passed to excludePlaylist to make the logic easier to follow.

## Requirements Checklist
- [X] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [X] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://jsbin.com/gejugat/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/master-playlist-controller.js [hunk 1]
```diff
@@ -1122,25 +1131,30 @@ export class MasterPlaylistController extends videojs.EventTarget {
...
-  excludePlaylist(error = {}, playlistExclusionDuration) {
+  excludePlaylist({
+    playlistToExclude = this.masterPlaylistLoader_.media(),
+    error = {},
+    playlistExclusionDuration
+  }) {
     // If the `error` was generated by the playlist loader, it will contain
     // the playlist we were trying to load (but failed) and that should be
     // excluded instead of the currently selected playlist which is likely
     // out-of-date in this scenario
```
Comment (gkatsev) [inline src/master-playlist-controller.js:1152]: is this comment still correct? At the very least, we won't be using the error.playlist here anymore.

**Review by gkatsev — APPROVED** [review]
One q around a comment. Otherwise, LGTM

File: src/master-playlist-controller.js (hunk 1 above)
Comment (gesinger) [reply src/master-playlist-controller.js:1152]: Nice catch! I'll remove the whole comment.

**Review by gkatsev — APPROVED** [review]

**Review by misteroneill — APPROVED** [review]

**Comment by misteroneill** [comment]
Rebased for @gesinger to resolve conflicts due to master/main renaming.

**Pull request merged**